.github/workflows/stacks-blockchain-api.yml

```diff
@@ -452,28 +459,55 @@ jobs:
             semantic-release-slack-bot
             @semantic-release/exec
 
-      - name: Build/Tag/Push Image
-        uses: docker/build-push-action@v1
+      - name: Docker Meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            blockstack/${{ github.workflow }}
+            hirosystems/${{ github.workflow }}
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern=${{ steps.semantic.outputs.new_release_version }}
+            type=sha
+
+      - name: Docker Standalone Meta
+        id: meta_standalone
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            blockstack/${{ github.workflow }}-standalone
+            hirosystems/${{ github.workflow }}-standalone
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern=${{ steps.semantic.outputs.new_release_version }}
+            type=sha
+
+      - name: Set up Docker Buildx
```

Comment:
Buildx is only needed with certain advanced docker usage features, like multi-platform builds or exporting a binary from a built container. You should be able to remove this step for both images.